viper/modules/rats/jrat.py

```diff
@@ -158,13 +169,14 @@ def parse_config(raw_config, enckey):
     config_dict["EncryptionKey"] = enckey.encode('hex')
     return config_dict
 
+
 def config(data):
     enckey, conf = get_parts(data)
-    if enckey == None:
+    if not enckey:
```

Comment:
This was actually auto suggest by the pep8 fixes. Also looks good on both py27 and py34:

```
>>> if 0 or "" or []:
...     print("something there")
... else:
...     print("nope")
...
nope
```